notebooks/geo/DEM_MapVisualization.ipynb

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
<p>Sorry I missed that earlier. I think we need to revise the way we select the data after we found him. Like to pretend we know how to use the PDS4 model from the beginning.</p><p> </p><p>I believe this request should work:</p><p>http://pds.nasa.gov/api/search/1/products/urn:nasa:pds:mars2020_rover_places:data_maps::7.0/members</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-PDS/search-api-notebook/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>